python/deltalake/writer.py

```diff
@@ -87,6 +87,7 @@ def write_deltalake(
     overwrite_schema: bool = False,
     storage_options: Optional[Dict[str, str]] = None,
     partition_filters: Optional[List[Tuple[str, str, Any]]] = None,
+    max_partitions: Optional[int] = None,
```

Comment:
Could you place this up near `max_open_files`, since it logically belongs with those options? Since we have the `*`, changing the order of the kwargs doesn't break backwards compatibility.

Reply:
Yes, my concern was around the backwards compat, didn't see the `*`.
Updated